### PR TITLE
hardcode 'managedIdentitiesDataPlaneAudienceResource' field of the azure runtime config to a dummy value.

### PR DESCRIPTION
**Pull request by machi1990**
Hardcode 'managedIdentitiesDataPlaneAudienceResource' field of the azure runtime config to a dummy value.
This is good enough for dev or int env.
Once templatize refactor and helm chart conversion is in place, the value will need to be parameterized.

### What this PR does

Jira: 
Link to demo recording:

### CODE REVIEW

File: cluster-service/deploy/openshift-templates/arohcp-service-template.yml [hunk 1]
```diff
@@ -367,6 +369,7 @@ objects:
       {
         "cloudEnvironment": "AzurePublicCloud",
         "tenantId": "${TENANT_ID}",
+        "managedIdentitiesDataPlaneAudienceResource": "https://dummy.org",
```
Comment (machi1990) [inline cluster-service/deploy/openshift-templates/arohcp-service-template.yml:372]: @janboll as discussed

**Review by janboll — DISMISSED** [review]

**Comment by miguelsorianod** [comment]
lgtm

File: cluster-service/deploy/openshift-templates/arohcp-service-template.yml [hunk 2]
```diff
@@ -357,6 +357,8 @@ objects:
             version_constraints:
               min_version: 4.11.0
 
+# TODO: parameterize managedIdentitiesDataPlaneAudienceResource after templatize refactor and helm chart conversion
+# so that the value can be configured in higher envs.
```
Comment (miguelsorianod) [inline cluster-service/deploy/openshift-templates/arohcp-service-template.yml:361]: Is there some jira to track this ?

Comment (machi1990) [reply cluster-service/deploy/openshift-templates/arohcp-service-template.yml:361]: None that I am aware about but @tony-schndr is progressing this is what I've been made aware of.

@tony-schndr @janboll do you know of any card ?

Comment (machi1990) [reply cluster-service/deploy/openshift-templates/arohcp-service-template.yml:361]: There was a general card about helm chat but not a specific one about it. 
I've created it under the same epic: ARO-12801

**Review by janboll — APPROVED** [review]

**Pull request merged**